packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -84,7 +84,7 @@ export const OdysseyThemeProvider = ({
   );
 
   return (
-    <div ref={contrastContainerRef}>
+    <div style={{ height: "inherit" }} ref={contrastContainerRef}>
```

Comment:
We need to memoize this style. Can we make this a Styled component like others as well?